pytest_django/plugin.py

```diff
@@ -239,7 +239,7 @@ def _get_boolean_value(x, name, default=None):
         return possible_values[x.lower()]
     except KeyError:
         raise ValueError(
-            "{} is not a valid value for {}. "
+            "{} is not a valid value for {}. "  # noqa: F502
             "It must be one of {}." % (x, name, ", ".join(possible_values.keys()))
```

Comment:
This doesn't work.  Earlier today I didn't really look at the code and just silenced the warning to unblock the PR.  But `{}` don't work in combination with `%s`.  This was introduced 4 years ago in commit https://github.com/pytest-dev/pytest-django/commit/3ec54d69e0ad14569c6d9e56d76ae7a29e772005#diff-87c0c2cb9262dd6c4384e140e51d8e64L165-R166 and nobody ever noticed, despite the TypeError being raised.

I'm going to change the `%` to `.format` which works with Python 2.7 and 3.2+, see https://pyformat.info/

Reply:
Hmm indeed, I stared at it when reviewing but missed that it used `%`. That makes more sense.